pkg/util/general/healthz.go

```diff
@@ -122,9 +154,35 @@ func RegisterReportCheck(name string, autoRecoverPeriod time.Duration) {
 		Message:           InitMessage,
 		AutoRecoverPeriod: autoRecoverPeriod,
 		Mode:              HealthzCheckModeReport,
+		shortTime:         false,
 	}
 }
 
+func RegisterShortTimeReportCheck(name string, autoRecoverPeriod time.Duration) {
+	healthzCheckLock.Lock()
+	defer healthzCheckLock.Unlock()
+
+	count := 1
+	current, ok := healthzCheckMap[HealthzCheckName(name)]
+	if ok {
```

Comment:
if healthz check already exists, just update the `autoRecoverPeriod` and return may be beter